dev/CommonStyles/FlipView_themeresources.xaml

```diff
@@ -117,7 +128,7 @@
                                     BorderThickness="{ThemeResource FlipViewButtonBorderThemeThickness}"
                                     BorderBrush="{ThemeResource FlipViewNextPreviousButtonBorderBrush}"
                                     contract7Present:CornerRadius="{TemplateBinding CornerRadius}"
-                                    contract7NotPresent:CornerRadius="{Binding Source={ThemeResource ControlCornerRadius}, Converter={StaticResource LeftCornerRadiusFilterConverter}}">
+                                    contract7NotPresent:CornerRadius="{TemplateBinding CornerRadius}">
```

Comment:
CornerRadius was introduced as property on controls in rs5 so to Template Bind it - we need to be at least on rs5. 
When contract7NotPresent you need to use ThemeResource for CornerRadius :)

Reply:
This was the only control that was not fixed in the previous commit, fixed now